Enterprise-CMCS/macpro-onemac
# Snyk + Dependabot + Cleanup...

Story: PRs in repo for snyk, dependabot, and some data cleanup
Endpoint: See github-actions bot comment

### Details
The PRs complaining about updates and what not should be satisfied... 

### Changes
- Package updates and audits as recommended by repo tools
- added deleteOutdatedItems to start a process to delete the many many records left over from the old SEA Tool stream
- updated processPackages to handle delete events
- added seed data to "be deleted" for testing purposes

### Test Plan
1. repo should deploy
2. tests should run 
3. application should work
4. run configure step to reload the test items to be deleted
5. verify that MA-23-0009 has SEATool#RAI# items
6. verify that MO-23-0010 has SEATool#State_Plan# items
7. use AWS console to run the d-20240105-deleteOutdatedItems script ( "reason" is the only required parameter and is only used in the Process item for knowing what has been scheduled)
8. Verify the process records got created
9. Processing happens in a schedule, can wait, or "trigger" it manually by using "Test" button on the processPackages lambda
10. Verify the RAI and State_plan records got deleted.
11. Verify the Process records finish and are removed.
